### PR TITLE
style: avatar updated

### PR DESCRIPTION
**Pull request by kvasdopil**
Fixing avatar style

Screenshot 2019-04-05 at 11 34 28

### CODE REVIEW

File: src/Atoms/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+import Text from '../Text';
 
 const StyledDiv = styled.div`
   background-color: ${p => p.theme.color.backgroundDark};
   width: ${p => p.theme.spacing.unit(8)}px;
   height: ${p => p.theme.spacing.unit(8)}px;
   border-radius: 50%;
-  line-height: ${p => p.theme.spacing.unit(8)}px;
+  line-height: ${p =>
+    p.theme.spacing.unit(8) - 1}px; /* FIXME: font is buggy, so baseline is 1px off */
```
Comment (JacobBlomgren) [inline src/Atoms/Avatar/Avatar.tsx]: 😭

File: src/Atoms/Avatar/Avatar.tsx [hunk 2]
```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Props } from './Avatar.types';
+import Text from '../Text';
```
Comment (shmidt-i) [inline src/Atoms/Avatar/Avatar.tsx]: Since we are importing other atom, let's move avatar to Molecules/

Comment (kvasdopil) [reply src/Atoms/Avatar/Avatar.tsx]: Done

File: src/Atoms/Avatar/Avatar.tsx (hunk 1 above)
Comment (kvasdopil) [reply src/Atoms/Avatar/Avatar.tsx]: I feel your pain bro, if you have a better solution I'm all ears.

**Pull request merged**